diff --git a/website/public/locales/en.json b/website/public/locales/en.json
index d110749..cd562c6 100644
--- a/website/public/locales/en.json
+++ b/website/public/locales/en.json
@@ -59,7 +59,9 @@
     "legend": "The encrypted message will be deleted automatically after",
     "optionOneHourLabel": "One Hour",
     "optionOneDayLabel": "One Day",
-    "optionOneWeekLabel": "One Week"
+    "optionOneWeekLabel": "One Week",
+    "optionTwoWeeksLabel": "Two Weeks",
+    "optionFourWeeksLabel": "Four Weeks"
   },
   "features": {
     "title": "Share Secrets Securely With Ease",
diff --git a/website/src/shared/Expiration.tsx b/website/src/shared/Expiration.tsx
index 135701a..2024300 100644
--- a/website/src/shared/Expiration.tsx
+++ b/website/src/shared/Expiration.tsx
@@ -1,60 +1,72 @@
 import { Controller, UseFormMethods } from 'react-hook-form';
 import { useTranslation } from 'react-i18next';
 import {
-  FormControl,
-  FormControlLabel,
-  FormLabel,
-  makeStyles,
-  Radio,
-  RadioGroup,
+    FormControl,
+    FormControlLabel,
+    FormLabel,
+    makeStyles,
+    Radio,
+    RadioGroup,
 } from '@material-ui/core';
 
 const useStyles = makeStyles({
-  radioGroup: {
-    justifyContent: 'center',
-  },
+    radioGroup: {
+        justifyContent: 'center',
+    },
 });
 
 export const Expiration = (props: { control: UseFormMethods['control'] }) => {
-  const { t } = useTranslation();
-  const classes = useStyles();
-  return (
-    <FormControl component="fieldset" margin="dense">
-      <FormLabel component="legend">{t('expiration.legend')}</FormLabel>
-      <Controller
-        rules={{ required: true }}
-        control={props.control}
-        defaultValue="3600"
-        name="expiration"
-        as={
-          <RadioGroup
-            row
-            classes={{
-              root: classes.radioGroup,
-            }}
-          >
-            <FormControlLabel
-              labelPlacement="end"
-              value="3600"
-              control={<Radio color="primary" />}
-              label={t('expiration.optionOneHourLabel')}
+    const { t } = useTranslation();
+    const classes = useStyles();
+    return (
+        <FormControl component="fieldset" margin="dense">
+            <FormLabel component="legend">{t('expiration.legend')}</FormLabel>
+            <Controller
+                rules={{ required: true }}
+                control={props.control}
+                defaultValue="3600"
+                name="expiration"
+                as={
+                    <RadioGroup
+                        row
+                        classes={{
+                            root: classes.radioGroup,
+                        }}
+                    >
+                        <FormControlLabel
+                            labelPlacement="end"
+                            value="3600"
+                            control={<Radio color="primary" />}
+                            label={t('expiration.optionOneHourLabel')}
+                        />
+                        <FormControlLabel
+                            labelPlacement="end"
+                            value="86400"
+                            control={<Radio color="primary" />}
+                            label={t('expiration.optionOneDayLabel')}
+                        />
+                        <FormControlLabel
+                            labelPlacement="end"
+                            value="604800"
+                            control={<Radio color="primary" />}
+                            label={t('expiration.optionOneWeekLabel')}
+                        />
+                        <FormControlLabel
+                            labelPlacement="end"
+                            value="1209600"
+                            control={<Radio color="primary" />}
+                            label={t('expiration.optionTwoWeeksLabel')}
+                        />
+                        <FormControlLabel
+                            labelPlacement="end"
+                            value="2419200"
+                            control={<Radio color="primary" />}
+                            label={t('expiration.optionFourWeeksLabel')}
+                        />
+                    </RadioGroup>
+                }
             />
-            <FormControlLabel
-              labelPlacement="end"
-              value="86400"
-              control={<Radio color="primary" />}
-              label={t('expiration.optionOneDayLabel')}
-            />
-            <FormControlLabel
-              labelPlacement="end"
-              value="604800"
-              control={<Radio color="primary" />}
-              label={t('expiration.optionOneWeekLabel')}
-            />
-          </RadioGroup>
-        }
-      />
-    </FormControl>
-  );
+        </FormControl>
+    );
 };
 export default Expiration;
